cpp/include/cudf/copying.hpp

```diff
@@ -143,27 +143,20 @@ std::unique_ptr<column> reverse(
  * A negative value `i` in the `scatter_map` is interpreted as `i+n`, where `n`
  * is the number of rows in the `target` table.
  *
- * @throws cudf::logic_error if `check_bounds == true` and an index exists in
```

Comment:
Perhaps the docs should say that behavior is undefined if out-of-bounds indices are passed? (for all versions of scatter that used to have a `check_bounds` option.

Reply:
@davidwendt had the same request :) Added.